localisation/decisions.csv

```diff
@@ -327,6 +327,8 @@ annex_south_gondor_title;Annex South Gondor;x
 annex_south_gondor_desc;Finally, we control all of South Gondor. It is time we abolish the fiefdom government, and merge it with the realm's government.;x
 accept_harnedan_title;The Peoples of Harondor;x
 accept_harnedan_desc;Many inhabitants of South Gondor are of both Gondorian and Haradrim descent - they are the sons of the Gondorian settlers who came to South Gondor in the past. When Gondor left the region, the settlers stayed, and were integrated into the local Haradrim society. They are rightfully descendants of Gondor, and should be recognized as such.;x
+claim_harnendor_title;Claim Harnendor;x
+claim_harnendor_desc;In the past, the territory south of the Harnen river known as N�nzayan was Gondorian, and it's culture quite similar to that of Harondor. With our renewed presence in the area, we can reassert our claim to this land.;x
```

Comment:
Nînzayan is the Haradrim name for Harnendor (the region)

Reply:
Should've clarified that it's the Haradrim who call it Nînzayan